scalability_and_performance/cnf-low-latency-tuning.adoc

```diff
@@ -48,10 +48,15 @@ include::modules/cnf-understanding-workload-hints.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
-* For information on using the Performance Profile Creator (PPC) to generate a performance profile, see xref:../scalability_and_performance/cnf-create-performance-profiles.adoc#cnf-create-performance-profiles[Creating a performance profile].
+* For information about using the Performance Profile Creator (PPC) to generate a performance profile, see xref:../scalability_and_performance/cnf-create-performance-profiles.adoc#cnf-create-performance-profiles[Creating a performance profile].
 
 include::modules/cnf-configuring-workload-hints.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+
+* For information about reducing CPU throttling for individual guaranteed pods, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#disabling-cpu-cfs-quota_{context}[Disabling CPU CFS quota].
```

Comment:
```suggestion
* For information about reducing CPU throttling for individual guaranteed pods, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#disabling-cpu-cfs-quota[Disabling CPU CFS quota].
```

Reply:
Yea funny I checked that against another section of the docs that had it set like this but as you state maybe to do with being in the same assembly.